src/main/java/sr/will/jarvis/command/CommandDefine.java

```diff
@@ -43,7 +44,7 @@ public void execute(Message message, String... args) {
 
         EmbedBuilder embed = new EmbedBuilder()
                 .setColor(Color.GREEN)
-                .setTitle(args[0], "https://www.urbandictionary.com/define.php?term=" + definition.list.get(0).word)
+                .setTitle(condenseArgs(args), "https://www.urbandictionary.com/define.php?term=" + definition.list.get(0).word)
```

Comment:
If you could change the condenseArgs call here to definition.list.get(0).word that would be appreciated.